hedera-node/hedera-app/src/testFixtures/java/com/hedera/node/app/fixtures/state/FakeState.java

```diff
@@ -66,7 +66,11 @@ public class FakeState implements State {
      */
     public FakeState addService(@NonNull final String serviceName, @NonNull final Map<String, ?> dataSources) {
         final var serviceStates = this.states.computeIfAbsent(serviceName, k -> new ConcurrentHashMap<>());
-        serviceStates.putAll(dataSources);
+        dataSources.forEach((k, b) -> {
+            if (!serviceStates.containsKey(k)) {
+                serviceStates.put(k, b);
+            }
+        });
```

Comment:
This slightly changes the behavior, yes? Instead of putting all data – i.e. overwriting any data already there – this only puts data that is new. Just making sure that's intended?

Reply:
This was a bug surfaced while testing. We should add any state only when state is not present.